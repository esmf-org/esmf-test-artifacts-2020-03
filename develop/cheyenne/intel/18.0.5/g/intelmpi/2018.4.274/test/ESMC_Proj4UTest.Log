build time -- 2022-01-18 06:21:17.219623
20220118 064758.868 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220118 064758.868 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220118 064758.868 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220118 064758.868 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220118 064758.868 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220118 064758.868 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220118 064758.868 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_05-30-gf84ebe0184
20220118 064758.868 INFO             PET0 ESMF library build date/time: "Jan 18 2022" "06:15:06"
20220118 064758.868 INFO             PET0 ESMF library build location : /glade/scratch/rlong/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220118 064758.868 INFO             PET0 ESMF_COMM                   : intelmpi
20220118 064758.868 INFO             PET0 ESMF_MOAB                   : enabled
20220118 064758.868 INFO             PET0 ESMF_LAPACK                 : enabled
20220118 064758.868 INFO             PET0 ESMF_NETCDF                 : enabled
20220118 064758.868 INFO             PET0 ESMF_PNETCDF                : disabled
20220118 064758.868 INFO             PET0 ESMF_PIO                    : enabled
20220118 064758.868 INFO             PET0 ESMF_YAMLCPP                : enabled
20220118 064758.868 INFO             PET0 Beginning Test, file ESMC_Proj4UTest.C, line 49
20220118 064758.868 INFO             PET0 NUMBER_OF_PROCESSORS 1
20220118 064758.868 INFO             PET0 PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
20220118 064758.868 INFO             PET0 Ending Test, file ESMC_Proj4UTest.C, line 123
20220118 064758.868 INFO             PET0 Finalizing ESMF
