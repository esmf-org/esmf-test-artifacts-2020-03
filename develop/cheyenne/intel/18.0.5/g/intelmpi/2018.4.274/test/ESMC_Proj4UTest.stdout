build time -- 2022-01-05 20:20:21.738556
Beginning Test, file ESMC_Proj4UTest.C, line 49
NUMBER_OF_PROCESSORS 1
PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
Ending Test, file ESMC_Proj4UTest.C, line 123
 PET 0 Test Elapsed Time  0.803000 msec.
